src/components/SpectralLineQuery/SpectralLineQueryComponent.tsx

```diff
@@ -197,12 +197,12 @@ export class SpectralLineQueryComponent extends React.Component<WidgetProps> {
 
     private handleFilter = () => {
         this.widgetStore.filter();
-        this.resultTableRef?.scrollToRegion(Regions.row(0));
+        setTimeout(() => this.resultTableRef?.scrollToRegion(Regions.row(0)), 20);
```

Comment:
@duidae do you think we should use a handle here so that we can `clearTimeout` of the existing scroll instruction?

Reply:
@veggiesaurus handle added, but the behavior seems the same before/after the timer handle added, a little bit unsure about why we need it?

Reply:
I suppose its not necessary. I was just worried that we would have multiple `handleFilter` functions resetting the scroll to row 0.